tests/integration_tests/test_chainermn.py

```diff
@@ -458,8 +464,7 @@ def _create_new_chainermn_trial(
 ) -> integration.chainermn.ChainerMNTrial:
 
     if comm.rank == 0:
-        trial_id = study._storage.create_new_trial(study._study_id)
-        trial = Trial(study, trial_id)
+        trial = study.ask()
```

Comment:
Minor simplification by using `ask` method. This way is also used in `tests/integration_tests/test_pytorch_distributed.py`.